src/mayara_pi.cpp

```
@@ -1522,6 +1558,9 @@ void mayara_pi::DestroyChartMenu() {
   m_chart_menu = nullptr;
   m_chart_menu_canvas = -1;
   p->Destroy();
+  // m_chart_menu_rect is already current -- every drag/resize wrote straight
+  // into it -- so closing is exactly when there is something new to persist.
+  if (m_chart_menu_user_moved || m_chart_menu_user_sized) SaveConfig();
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)